tests/bip32.py

```diff
@@ -24,7 +24,6 @@
 from ecdsa.curves import SECP256k1
 from ecdsa.ellipticcurve import INFINITY, Point
 from ecdsa.util import number_to_string, string_to_number
-
 from trezorlib import messages, tools
```

Comment:
generally we want "trezorlib" to always be in the "1st party" section or whatever it is called, i.e., separated from click / pytest / ecdsa / etc.
there should be no diff here and in 90 % of `tests/`